addons/mail/static/src/core/common/thread.js

```diff
@@ -153,6 +156,18 @@ export class Thread extends Component {
         onMounted(async () => {
             this.lastJumpPresent = this.props.jumpPresent;
             await this.threadService.fetchNewMessages(this.props.thread);
+            if (this.props.thread.id === "inbox") {
+                const [user] = await this.orm.read(
+                    "res.users",
+                    [this.store.user.user?.id],
+                    ["partner_id", "notification_type"]
+                );
```

Comment:
Do you have a scenario where `store.user.user?.id` is falsy and you have access to inbox?

I don't mind robust code that works in all environments, but I would expect to not make the RPC, as to not provide a non-id which might lead to a traceback